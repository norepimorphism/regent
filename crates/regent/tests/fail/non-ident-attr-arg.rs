fn main() {
    // Path (leading colon).
    #[regent::bitwise(::foo = 0)]
    struct A;
    // Path (2 segments).
    #[regent::bitwise(foo::bar = 0)]
    struct B;
    // Path (3 segments).
    #[regent::bitwise(foo::bar::qux = 0)]
    struct C;
}
