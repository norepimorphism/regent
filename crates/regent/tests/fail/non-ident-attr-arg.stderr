error: argument path must be an identifier
 --> tests/fail/non-ident-attr-arg.rs:3:23
  |
3 |     #[regent::bitwise(::foo = 0)]
  |                       ^^^^^

error: argument path must be an identifier
 --> tests/fail/non-ident-attr-arg.rs:6:23
  |
6 |     #[regent::bitwise(foo::bar = 0)]
  |                       ^^^^^^^^

error: argument path must be an identifier
 --> tests/fail/non-ident-attr-arg.rs:9:23
  |
9 |     #[regent::bitwise(foo::bar::qux = 0)]
  |                       ^^^^^^^^^^^^^
